### PR TITLE
Update Pit of Saron

### PR DESCRIPTION
**Pull request by talamortis**
Fix for Pit of Saron as player enters the dungon.

### CODE REVIEW

File: src/scripts/Northrend/FrozenHalls/PitOfSaron/instance_pit_of_saron.cpp [hunk 1]
```diff
@@ -77,6 +77,8 @@ class instance_pit_of_saron : public InstanceMapScript
         void OnPlayerEnter(Player* plr)
         {
             instance->LoadGrid(LeaderIntroPos.GetPositionX(), LeaderIntroPos.GetPositionY());
+            if (Creature* c = instance->GetCreature(GetData64(DATA_LEADER_FIRST_GUID)))
+				     c->AI()->SetData(DATA_START_INTRO, 0);
```
Comment (ShinDarth) [inline src/scripts/Northrend/FrozenHalls/PitOfSaron/instance_pit_of_saron.cpp]: a tab should be 4 spaces, can you fix your code indentation? thank you

**Comment by ShinDarth** [comment]
@talamortis 

https://github.com/azerothcore/azerothcore-wotlk/pull/78/files

how it is now:

image

how it should be:

image

the if aligned with the previous line, the content of the if aligned with 4 spaces

I suggest to set your code editor config  "tab size" to 4 spaces

**Comment by Yehonal** [comment]
@ShinDarth seems you attached the same screenshot

**Comment by ShinDarth** [comment]
@Yehonal yes, my mistake, edited

**Comment by DevRival** [comment]
I test, good work!!

**Comment by talamortis** [comment]
thanks :)

**Pull request merged**